README.md

```diff
@@ -2,22 +2,36 @@
 
 # Terraform Plugin Framework
 
-terraform-plugin-framework is an experimental module with no backwards compatibility guarantees, as indicated by its version 0 SemVer releases. We are mindful of our consumers and will strive to either maintain backwards compatibility or offer low-effort upgrade paths to provider developers, but our compatibility guarantees on this module are weaker than the [terraform-plugin-sdk](https://github.com/hashicorp/terraform-plugin-sdk) module. The purpose of this repository is to get real world experience with the design and implementation of this code before it is bound by the SDK's compatibility guarantees. This module is intentionally limited in its lifespan, and will become the next major version of the terraform-plugin-sdk module once were confident in its design and implementation, and this repository will be archived. When this happens, we will provide an upgrade path from users of the terraform-plugin-framework repository to the terraform-plugin-sdk release.
+terraform-plugin-framework is a module for building Terraform providers built on [terraform-plugin-go](https://github.com/hashicorp/terraform-plugin-go). It aims to provide as much of the power, predictability, and versatility of terraform-plugin-go as it can while abstracting away implementation details and repetitive, verbose tasks.
```

Comment:
> building Terraform providers built on [terraform-plugin-go](https://github.com/hashicorp/terraform-plugin-go)
```suggestion
terraform-plugin-framework is a module for building Terraform providers. It is built on [terraform-plugin-go](https://github.com/hashicorp/terraform-plugin-go). It aims to provide as much of the power, predictability, and versatility of terraform-plugin-go as it can while abstracting away implementation details and repetitive, verbose tasks.
```
Suggest reversing the order @